lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -64,6 +64,12 @@ static llvm::cl::opt<bool> SkipFoldingAutoDiffFunctionExtraction(
     "differentiation-skip-folding-autodiff-function-extraction",
     llvm::cl::init(true));
 
+/// This flag is used to disable `autodiff_function_extract` instruction folding
```

Comment:
Please update this comment.

Reply:
Will update to `This flag is used to enable full JVP generation.